modules/contact/doc/content/modules/contact/index.md

```diff
@@ -270,3 +270,38 @@ The recommended PETSc options for use with `NodeFaceConstraint` based contact ar
 ## Objects, Actions, and Syntax
 
 !syntax complete groups=ContactApp level=3
+
+## Scaling effects
+
+The effects of scaling on non-linear solve convergence are shown below for
+different kinds of contact formulations. The results are based on the input files for
+[RANFS](/ranfs-and-scaling/bouncing-block-ranfs.i),
+[kinematic](/bouncing-block-kinematic.i), and
+[tangential penalty](/bouncing-block-tan-pen.i),
+running with two processes.
+
+| Scheme | Cumulative nonlinear iterations | Cumulative linear iterations |  Condition number | Scaling Factor |
```

Comment:
Is 'Scaling Factor' the number that auto scaling computed? Was the same factor applied to all displacement variables?

Reply:
Yes and yes. This PR adds the ability to group variable together for determination of scaling factors. All input files using automatic scaling have the displacement variables grouped.